project/Dependencies.scala

```diff
@@ -25,13 +25,14 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-persistence-tck" % AkkaVersion % Test,
     "com.typesafe.akka" %% "akka-stream-testkit" % AkkaVersion % Test,
     "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % Test,
-    "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion,
+    "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion % Provided,
     "com.typesafe.slick" %% "slick" % SlickVersion,
     "com.typesafe.slick" %% "slick-hikaricp" % SlickVersion,
     "org.scalatest" %% "scalatest" % ScalaTestVersion % Test) ++ JdbcDrivers.map(_ % Test)
 
   val Migration: Seq[ModuleID] = (Seq(
     "org.scalatest" %% "scalatest" % ScalaTestVersion,
+    "com.typesafe.akka" %% "akka-persistence-query" % AkkaVersion,
```

Comment:
See [this comment](https://github.com/SwissBorg/akka-persistence-postgres/pull/228/files#r774475643).

Reply:
yep good catch, changed